src/harness/fourslashRunner.ts

```diff
@@ -4,7 +4,8 @@
 
 const enum FourSlashTestType { 

```

Comment:
It woudl be great if you documented.  Explain that fourslash always tests the LS api.  And, we can take advantage of that to test the shim and server layers by wrapping them so tehy expose the LS api.  etc. etc.